community/front-end/ofe/website/ghpcfe/cluster_manager/filesystem.py

```diff
@@ -56,7 +56,7 @@ def write_filestore_yaml(fs: GCPFilestoreFilesystem, target_dir: Path) -> None:
     id: {fs.name}
     settings:
       filestore_share_name: {export_name[1:]}
-      network_name: {fs.vpc.cloud_id}
+      network_id: projects/{project_id}/global/networks/{fs.vpc.cloud_id}
```

Comment:
This is not blocking the PR, but in the future, I would have maybe suggested that you included a pre-existing-vpc module in this blueprint and then just "use: network" in the filestore. Doing so would have prevented this type of error.